Source/MediaStorage.swift

```diff
@@ -236,7 +239,52 @@ public class MediaStorage {
             completionHandler(MediaContent(data: imageData), MediaStorageError(statusCode: nil, message: nil))
         }
     }
-
+    
+    public func addMeta(mediaId mediaId: String, userMeta: Dictionary<String, String>, completionHandler: MediaStorageError -> Void) {
+        if accessToken == nil {
+            completionHandler(
+                MediaStorageError(statusCode: nil, message: "wrong usage: use the connect method to get an access token.")
+            )
+            return
+        }
+        
+        for (key, value) in userMeta {
+            let userMetaKey = replaceUserMeta(key)
+            if userMetaKey.isEmpty || !isValidValue(value) {
+                completionHandler(
+                    MediaStorageError(statusCode: nil, message: "invalid parameter: \(userMeta)")
```

Comment:
I think it'd be easier to read the error message if it has the single key&value set which is invalid, not the whole sets in `userMeta`.